### PR TITLE
Review print

### PR DESCRIPTION
**Pull request by bobbus**

### CODE REVIEW

**Comment by bobbus** [comment]
Still to do

**Comment by bobbus** [comment]
@lcoq : ready for review

File: app/components/reviews-filters.js [hunk 1]
```diff
@@ -8,6 +8,14 @@ function elementIsOrIsIn($element, $container) {
   return $element.is($container) || $element.closest($container).length;
 }
 
+function allSelectedProperty(allProperty, selectedProperty) {
+  return Ember.computed(allProperty, selectedProperty, function() {
+    const total = get(this, allProperty + '.length');
+    const selected = get(this, selectedProperty + '.length');
+    return (total == selected);
```
Comment (lcoq) [inline app/components/reviews-filters.js]: I think JSHint does not like non strict equal operator

File: app/models/entry-group-by-client-and-project-list.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ export default EntryGroup.extend({
   entries: null,
   groups: null,
 
+  multipleGroups: Ember.computed('groups', function() {
+    const groupsLength = get(this, 'groups.length');
+    return groupsLength > 1;
+  }),
```
Comment (lcoq) [inline app/models/entry-group-by-client-and-project-list.js]: [`Ember.computed.gt('groups.length', 1)`](http://emberjs.com/api/classes/Ember.computed.html#method_gte)

File: app/components/show-entry.js [hunk 3]
```diff
@@ -24,7 +24,11 @@ export default Ember.Component.extend({
     'rounding:disabled'
   ],
 
-  showDate: true,
+  groupedBy: null,
+  groupedByDay: Ember.computed.equal('groupedBy', 'day'),
+  groupedByProject: Ember.computed.equal('groupedBy', 'project'),
```
Comment (lcoq) [inline app/components/show-entry.js]: no need to change but I usually prefix boolean with `is` => `isGroupedByDay`

**Comment by bobbus** [comment]
@lcoq : all fixed

**Pull request merged**